docker/testStorageMysql/start.sh

```diff
@@ -13,11 +13,10 @@ MYSQL_CONTAINER_PORT=3306
 #MYSQL_PASSWORD=$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 32 | sed 1q)
 
 
-is_container_created=$(docker ps -a | grep ${MYSQL_CONTAINER_NAME})
-is_container_running=$(docker ps | grep ${MYSQL_CONTAINER_NAME})
+is_container_running=$(docker inspect --format="{{ .State.Running }}" ${MYSQL_CONTAINER_NAME} 2> /dev/null)
```

Comment:
previous approach looks more readable and clear to me. Does this change fixes or improves anything?